### PR TITLE
Set mouse cap

### PR DESCRIPTION
**Pull request by jwlo44**
adding a public method so other classes can tell the moving camera whether or not to capture mouse movement

### CODE REVIEW

File: Manbil/Input/MovingCamera.h [hunk 1]
```diff
@@ -26,17 +26,21 @@ class MovingCamera : public Camera
...
+		: Camera(pos, forward, up, lockUp), pressedToggleKey(false),
           MoveSpeed(moveSpd), RotSpeed(rSpeed),
           mouseTarget(100, 100) { }
 
 
     bool IsMouseCapped(void) const { return capMouse; }
+	// turn mouse capture on or off
+	void SetMouseCapped(bool isCapped) {
+		capMouse = isCapped;
+	}
```
Comment (heyx3) [inline Manbil/Input/MovingCamera.h]: Could you change this to be a one-liner?

Also, even if you wanted to make this multi-line, keep in mind that the braces style for this framework is on their own line.

:bird:

Comment (heyx3) [inline Manbil/Input/MovingCamera.h]: Also also, the comment is unnecessary; it's just a recording of the actual function name.

Comment (jwlo44) [inline Manbil/Input/MovingCamera.h]: okay, I changed it

**Comment by heyx3** [comment]
:bird: :bird: :bird:

**Pull request merged**